### PR TITLE
Add checkbox for excluding anonymous chat

### PR DESCRIPTION
**Pull request by evanlucas**
R=@suitupalex

### CODE REVIEW

File: ampersand-chart.js [hunk 1]
```diff
@@ -484,6 +484,18 @@
       filterPersonnel.append('h6')
         .text('By agent/team:');
 
+      var filterAnon = filterWindow.append('section')
+        .attr('class', 'ampersand-graph-filter-widget ampersand-graph-exclude-anonymous');
+
+      var anonId = 'exclude_anonymous_' + this.model.title.replace(/\s/g, '_');
```
Comment (umamialex) [inline ampersand-chart.js]: Mind making this ID kebabcase and prefixed with `ampersand-graph`?

Comment (evanlucas) [inline ampersand-chart.js]: sure

**Comment by evanlucas** [comment]
Alright, updated :]

**Comment by umamialex** [comment]
LGTM. Just needs a version bump.

**Comment by evanlucas** [comment]
Done

**Pull request merged**